GPU/Common/FragmentShaderGenerator.cpp

```diff
@@ -558,10 +559,19 @@ bool GenerateFragmentShader(const FShaderID &id, char *buffer, const ShaderLangu
 						WRITE(p, "  vec4 t = tex2D(tex, %s.xy)%s;\n", texcoord, bgraTexture ? ".bgra" : "");
 					}
 				} else {
-					if (doTextureProjection) {
-						WRITE(p, "  vec4 t = %sProj(tex, %s);\n", compat.texture, texcoord);
+					if (texture3D) {
+						WRITE(p, "  float bias = pow(u_mipBias, 1.0);\n");
```

Comment:
note to self, remove, was experimenting to see if a curve affected the perception of smoothness.

Additionally, this won't combine with shader depal currently, should probably lock it out in the shaderid level.